docs/applier.md

```diff
@@ -3,7 +3,7 @@
 The file [templateprocessor](../pkg/templateprocessor) contains an number of methods allowing you to render template yamls. 
```

Comment:
@itdove should we change "yamls" to YAML files? 

Please make that change if you agree?

I added a comma below.

Reply:
yes